[skip-changelog] Add explanation on how to change the monitor configuration to FAQ (#2247)
arduino/arduino-cli: docs/FAQ.md

There are many excellent serial terminals to chose from. On Linux or macOS, you may already have [screen][screen]
installed. On Windows, a good choice for command line usage is Plink, included with [PuTTY][putty].

## How to change Monitor Configuration?

[Configuration parameters][configuration parameters] of the monitor can be obtained by executing the following commad:

`$ arduino-cli monitor -p <port> --describe`

These parameters can be modified by passing a list of `<key>=<desiredValue>` pairs to the `--config` flag. For example,

> to change the baud rate of the serial monitor to 4800, the command would be the following:

Reviewer: ```suggestion
when using a serial port, the monitor baud rate can be set to 4800 with the following command:
```

Adjust wording to move the association of the `serial` protocol from the monitor to the port. This is intended to communicate the concept that the monitor is a general purpose tool that can be used with any arbitrary communication channel rather than being specific to serial ports.

[screen]: https://www.gnu.org/software/screen/manual/screen.html
[putty]: https://www.chiark.greenend.org.uk/~sgtatham/putty/
[monitor command]: commands/arduino-cli_monitor.md

> [configuration parameters]: https://arduino.github.io/arduino-cli/dev/pluggable-monitor-specification/#describe-command

Reviewer: ```suggestion
[configuration parameters]: pluggable-monitor-specification.md#describe-command
```

Use relative link so that the target of the link will be of the same instance of the documentation as the page that was the source of the link.

Link to source file so that it will be functional when viewing the Markdown file directly in addition to when using the documentation website (MkDocs automatically converts the link to point to the appropriate HTML page in the generated website).
